src/backend/joanie/core/api/admin.py

```diff
@@ -550,3 +550,28 @@ def destroy(self, request, *args, **kwargs):
             ) from error
 
         return super().destroy(request, *args, **kwargs)
+
+
+class EnrollmentViewSet(
```

Comment:
This viewset could be the first one in a dedicated module to start splitting this one later ?